overwrite/config/application.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'boot'
+
+require 'rails/all'
+
+# Require the gems listed in Gemfile, including any gems
+# you've limited to :test, :development, or :production.
+Bundler.require(*Rails.groups)
+
+module Railsapp
+  class Application < Rails::Application
+    # Initialize configuration defaults for originally generated Rails version.
+    config.active_record.schema_format = :sql
```

Comment:
I also removed a line here (load defaults for a specific version) I guess we can't use that as we're testing against multiple versions of rails.